src/__test__/helper.test.js

```diff
@@ -0,0 +1,21 @@
+const { getDataFromLocalStorage, updateLocalStorage } = require('../helper');
```

Comment:
- [ ] Please change this to es6 imports

Reply:
This is not mendatory.

Reply:
This project is already approved only tests need to be checked.

Reply:
Kindly note that using es6 syntax is mandatory @talhawaqar